src/mlpack/methods/adaboost/adaboost_main.cpp

```diff
@@ -205,6 +206,10 @@ static void mlpackMain()
       trainingData.shed_row(trainingData.n_rows - 1);
     }
 
+    // Sanity check on data and labels
+    util::CheckSameSizes(trainingData, labelsIn, 
+            "AdaBoostModel::AdaBoostModel()");
```

Comment:
Should be tabbed twice (4 spaces), see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping for more details.